Mathlib/CategoryTheory/Equivalence.lean

```diff
@@ -320,14 +325,14 @@ def funInvIdAssoc (e : C ≌ D) (F : C ⥤ E) : e.functor ⋙ e.inverse ⋙ F 
   (Functor.associator _ _ _).symm ≪≫ isoWhiskerRight e.unitIso.symm F ≪≫ F.leftUnitor
 #align category_theory.equivalence.fun_inv_id_assoc CategoryTheory.Equivalence.funInvIdAssoc
 
-@[simp]
+@[simp, reassoc (attr := simp)]
```

Comment:
This one and the three lemmas don't have compositions in their statement, so they don't need the `reassoc` attribute.